blocks/multiplatform/videos/video-block/video-block.module.css

```diff
@@ -28,8 +28,22 @@
 }
 
 .videoPlayer {
-    border-radius: 24px;
+    --video-player-ratio: 16 / 9;
+    --video-border-radius: 24px;
+
     overflow: hidden;
     width: 100%;
     margin: 48px 72px 72px;
-}
\ No newline at end of file
+}
+
+.videoPlayer > div {
+    padding-bottom: calc(1% * (100 / (var(--video-player-ratio)))) !important;
```

Comment:
@nikpachoo, looks like original video is not 16/9. And it was cropped in original, could you double check? 🙏